server/src/services/person.service.spec.ts

```diff
@@ -437,6 +437,36 @@ describe(PersonService.name, () => {
     });
   });
 
+  describe('unassignFace', () => {
+    it('should unassign a face', async () => {
+      personMock.getFaceById.mockResolvedValueOnce(faceStub.face1);
+      accessMock.person.checkOwnerAccess.mockResolvedValue(new Set([personStub.noName.id]));
+      accessMock.person.checkFaceOwnerAccess.mockResolvedValue(new Set([faceStub.face1.id]));
+      personMock.reassignFace.mockResolvedValue(1);
+      personMock.getRandomFace.mockResolvedValue(null);
+      personMock.getFaceById.mockResolvedValueOnce(faceStub.unassignedFace);
+
+      await expect(sut.unassignFace(authStub.admin, faceStub.face1.id)).resolves.toStrictEqual(
+        mapFaces(faceStub.unassignedFace, authStub.admin),
+      );
```

Comment:
I think it would make sense to also expect some of the repo methods to have been called. Specifically that `requirePermission` gets called for `PERSON_WRITE`. Also, I would like to see a test without a personId, so that you don't run into that if but instead run into the `createNewFeaturePhoto` method. Basically try to cover the relevant branches.